dev/packages/alpha/cisco-0.0.2/dataset/asa/manifest.yml

```diff
@@ -0,0 +1,52 @@
+title: Cisco asa logs
```

Comment:
I assume that "ASA" should be written in caps. If so, then please adjust all titles and descriptions.